include/laser_filters/scan_shadows_filter.h

```diff
@@ -1,6 +1,7 @@
 /*
- * Copyright (c) 2008 Radu Bogdan Rusu <rusu@cs.tum.edu>
+ * Software License Agreement (BSD License)
```

Comment:
Let's leave radu in the copyright line explicitly. How about `Copyright (c) 2008 Radu Bogdan Rusu <rusu@cs.tum.edu> and other laser_filters authors`

Reply:
Okay, done